networks/mainnet.json

```diff
@@ -1,823 +1,855 @@
+        "StdComptrollerG4": 10348750,
+        "cYFI": 12848198,
+        "BravoDelegator2": 14181018,
+        "cTUSDDelegate": 11008130,
+        "StdComptrollerG5": 10809458,
+        "cREP": 7710755,
+        "Base200bps_Slope2000bps_Jump8000bps_Kink90": 9847407,
+        "cWBTC2Delegate": 11960638,
+        "Base200bps_Slope222bps_Kink90_Jump40": 8983555,
+        "wbtc2_irm": 11998859,
+        "cZRX": 7710733,
+        "cWBTC": 8163813,
+        "cLINK": 12286030,
+        "cYFIDelegate": 12654137,
+        "DSR_Updateable": 10504587
+    },
+    "PriceFeed": {
+        "description": "ChainLink + Uniswap Anchored View",
+        "address": "0x6D2299C48a8dD07a872FDd0F8233924872Ad1071"
+    },
+    "PriceOracleProxy": {
+        "description": "Price Oracle Proxy",
+        "cETH": "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5",
+        "cUSDC": "0x39AA39c021dfbaE8faC545936693aC917d5E7563",
+        "cSAI": "0xF5DCe57282A584D2746FaF1593d3121Fcac444dC",
+        "cDAI": "0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643",
+        "cUSDT": "0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9",
+        "address": "0xDDc46a3B076aec7ab3Fc37420A8eDd2959764Ec4"
+    },
+    "Maximillion": {
+        "description": "Maximillion",
+        "cEtherAddress": "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5",
+        "address": "0xf859A1AD94BcF445A406B892eF0d3082f4174088"
+    },
+    "CompoundLens": {
+        "name": "CompoundLens",
+        "contract": "CompoundLens"
+    },
+    "Unitroller": {
+        "description": "Unitroller",
+        "address": "0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B"
+    },
+    "Reservoir": {
+        "name": "Reservoir",
+        "contract": "Reservoir"
+    },
+    "Comptroller": {
+        "StdComptroller": {
+            "address": "0x62F18C451af964197341d3c86D27e98C41BB8fcC",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        },
+        "StdComptrollerG2": {
+            "address": "0xf592eF673057a451c49c9433E278c5d59b56132c",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        },
+        "StdComptroller_2_6": {
+            "address": "0x97BD4Cc841FC999194174cd1803C543247a014fe",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        },
+        "StdComptrollerG3": {
+            "address": "0x9D0a0443FF4bB04391655B8cD205683d9fA75550",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        },
+        "StdComptrollerG4": {
+            "address": "0xAf601CbFF871d0BE62D18F79C31e387c76fa0374",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        },
+        "StdComptrollerG5": {
+            "address": "0x7b5e3521a049C8fF88e6349f33044c6Cc33c113c",
+            "contract": "Comptroller",
+            "description": "Standard Comptroller Impl"
+        }
```

Comment:
Other changes lgtm but did you have to modify this whole file? 😬

Reply:
Cleaned mainnet.json to match prev structure of how we update, however reverting mainnet-abi.json would bring it out of sync with how the current state of protocol on-chain. 

Should I revert mainnet-abi.json to match the current state of master?

Reply:
Nice, thank you! I wonder what actually changed in `mainnet-abi.json` besides the new abi?

Reply:
IIRC the existing `mainnet-abi.json` does not handle GovernorBravo's structure correctly with respect to .scen files, which is why @arr00 recommended using this ABI instead of the existing ABI. 

I'm not as familiar with the specific changes to the ABI, but can dive deeper/remove it from this PR if you think we should identify what actually changed. I was running into significant issues testing with the existing ABI, which is why I included it in this PR, so that future devs wouldn't experience similar issues with the ABI. LMK what you think is best!